src/library_glfw.js

```diff
@@ -981,14 +981,9 @@ var LibraryGLFW = {
       if (!win) return;
 
       if (GLFW.active.id == win.id) {
-        if (width == screen.width && height == screen.height) {
-          Browser.requestFullscreen();
-        } else {
-          Browser.exitFullscreen();
           Browser.setCanvasSize(width, height);
           win.width = width;
           win.height = height;
```

Comment:
Can you fix the indentation of this block?

Reply:
Done. Sorry.